### PR TITLE
Docker: consider very high memory limit as "unlimited"

### PR DESCRIPTION
**Pull request by dperpeet**

### CODE REVIEW

**Comment by dperpeet** [comment]
Also check for docker to have finished loading before trying to access a formatting function (led to "Oops" if accessing page too quickly after login due to delayed loading).

File: pkg/shell/cockpit-docker.js [hunk 1]
```diff
@@ -453,8 +453,8 @@ function render_container (client, $panel, filter_button, prefix, id, container,
     if (container.State && container.State.Running) {
         cputext = format_cpu_usage(container.CpuUsage);
 
-        memuse = container.MemoryUsage || 0;
-        memlimit = container.MemoryLimit || 0;
+        memuse = container.MemoryUsage;
+        memlimit = container.MemoryLimit;
```
Comment (stefwalter) [inline pkg/shell/cockpit-docker.js]: This change doesn't seem to be related. Could you explain it in the commit message?

Comment (dperpeet) [inline pkg/shell/cockpit-docker.js]: Explained: It's an unnecessary initialization of variables to default values. This makes it more difficult in the rest of the script to identify if the values have been provided by the Docker API / OS or set by Cockpit.
